pom.xml

```diff
@@ -69,7 +69,7 @@
         <!-- Properties below are set in this file because they are used
              in the BOM as well as other POMs (build-parent/pom.xml, docs/pom.xml, ...) -->
         <jacoco.version>0.8.12</jacoco.version>
-        <kubernetes-client.version>6.13.0</kubernetes-client.version> <!-- Please check with Java Operator SDK team before updating -->
+        <kubernetes-client.version>6.13.1</kubernetes-client.version> <!-- Please check with Java Operator SDK team before updating -->
```

Comment:
I think is better to wait for 6.13.2

Reply:
What's the reason for that?>

Reply:
I'm not sure we'll release 6.13.2 soon enough.
This patch already contains other much critical fixes (logs are currently bloated for example).